CODEOWNERS

```diff
@@ -89,7 +89,7 @@ Kconfig*                                  @tejlmand
 /modules/                                 @tejlmand
 /modules/mcuboot/                         @nvlsianpu
 /modules/cjson/                           @simensrostad @plskeggs @sigvartmh
-/samples/sensor/bh1749/                   @wlgrd
+/samples/sensor/bh1749/                   @jfischer-no
```

Comment:
What is the reason for this change?

Reply:
Since I am not an active contributor to nrfconnect, @carlescufi suggested to put you as code owner for this library.